src/shared/callbacks.rs

```diff
@@ -128,20 +128,20 @@ pub struct Callbacks {
 }
 
 impl Callbacks {
-    pub fn register_buckets_callback(&mut self, f: Function) {
-        self.buckets = Some(Callback::new(f));
+    pub fn register_buckets_callback(&mut self, _f: Function) {
```

Comment:
This looks good. I think a comment explaining why these functions don't do anything with a link to the original issue would be helpful.